scripts/report-bundle-size.js

```diff
@@ -1,21 +1,33 @@
 #!/usr/bin/env node
 /* eslint-env node */
-
+const fs = require('fs')
 const path = require('path')
 const childProc = require('child_process')
 
 /**
  * Send bundle size stats to Datadog
  */
 const main = () => {
-    const retailReactAppStats = require(path.join(path.resolve(''), 'packages', 'template-retail-react-app', 'build', 'stats.json'))
-
-    const bundles = retailReactAppStats.assets
-    bundles.forEach((bundle) => {
-        const metric = `mobify_platform_sdks.bundle_size_byte`
-        const value = bundle.size
-        childProc.spawnSync('dog', ['metric', 'post', metric, value, '--host', bundle.name])
-        console.log(`${metric} ${value} --host ${bundle.name}`)
+    const buildDir = path.join(path.resolve(''), 'packages', 'pwa', 'build')
+    fs.readdir(buildDir, (err, files) => {
+        if (err)
+            console.log(err);
```

Comment:
```suggestion
            console.error(err);
```

Reply:
I think we can also take this error handling further. How about making it so that it would fail CI? 

```js
console.error(err)
process.exit(1)
```
Inspired by https://github.com/SalesforceCommerceCloud/pwa-kit/blob/271e82c93475b9b34a1bce5ac4ba46f88819370a/scripts/check-dependencies.js#L275